src/pages/EventPage/components/MetaInfo/MetaInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { MetaInfoGroup, MetaInfoLabel, MetaInfoValue } from './styled';
+import {
+  formatDateInterval,
+  formatPersonName,
+} from '../../../../utils/formatting';
+import { Event } from '../../../../types/domain';
```

Comment:
Is there any way we can make thesee imports relative to the root/src of the project? Lots of these relative imports can be a pain to deal with if we end up wanting to reorganize folder structure and move things. For example, say we wanted to move this module into a different folder with some other distance from the root, then we would have to change the imports. But if the imports were relative to the root, then they would all stay the same.

Reply:
Yes please 😩! Webpack and tsconfig allow us to define path aliases, however, `create-react-app` doesn't allow us to use them 😭 - since there is some issues regarding this when building multiple packages (not too clear on this myself). It is, however, possible to define the basepath to be `src`, which would allow us to define imports relative to the `src` directory of the project, e.g. `src/utils/formatting`. It's a bit uglier than path aliases, but we can probably go for that in another PR.

Reply:
See 80bf37b in #31.